.github/workflows/ci.yml

```diff
@@ -202,14 +202,29 @@ jobs:
           node-version: "20"
           cache: "pnpm"
 
+      - name: Install stable Chromium
+        run: |
+          set -euo pipefail
+          CHROME_VERSION=$(curl -s https://googlechromelabs.github.io/chrome-for-testing/last-known-good-versions.json | jq -r '.channels.Stable.version')
+          DOWNLOAD_URL="https://storage.googleapis.com/chrome-for-testing-public/${CHROME_VERSION}/linux64/chrome-linux64.zip"
+          INSTALL_DIR="${RUNNER_TEMP}/chrome-stable"
```

Comment:
can save a bit of time on all tests by caching this dir in github actions cache but not critical for now (using OS, arch, and chrome version in cache key)